apps/ui/src/networks/starknet/index.ts

```diff
@@ -32,7 +32,7 @@ export const METADATA: Partial<Record<NetworkID, Metadata>> = {
     baseNetworkId: 'eth',
     rpcUrl: `https://starknet-mainnet.infura.io/v3/${import.meta.env.VITE_INFURA_API_KEY}`,
     ethRpcUrl: `https://mainnet.infura.io/v3/${import.meta.env.VITE_INFURA_API_KEY}`,
-    apiUrl: 'https://api.snapshot.box',
+    apiUrl: 'https://api-1.snapshot.box',
```

Comment:
Is this on purpose?

Reply:
Yes, this way it will use staging API, otherwise we can do without this change, but it will require new deployment (so changes on DigitalOcean, Box domain etc.), we can do either way depending on your availability to handle new deployment.

Reply:
Actually this is not needed as it doesn't matter what API we use on UI, api just needs to report it to mana and it's enough that one of those APIs will be resynced.